website/docs/d/schduled_time.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_scheduled_time"
+sidebar_current: "docs-azurerm-datasource-scheduled-time"
+description: |-
+  Get information about the specified scheduled time.
+---
+
+# azurerm\_scheduled\_time
+
+Use this data source to access the properties of an existing Scheduled Time.
+
+## Example Usage
+
+```hcl
+data "azurerm_scheduled_time" "test" {
+        "frequency" = "Day"
+        "hour" = "14"
+        "minute" = "25"
+        "minimum_delay_from_now_in_minutes" = "6"
```

Comment:
we don't need the quotes around the `key` part of these values, or where they're Int's - so I think we can make this:

```
frequency = "Day"
hour = 14
minute = 25
```

Reply:
Done!